google-cloud-clients/google-cloud-firestore/pom.xml

```diff
@@ -108,6 +108,18 @@
       <classifier>testlib</classifier>
       <scope>test</scope>
     </dependency>
+      <dependency>
+          <groupId>com.fasterxml.jackson.core</groupId>
+          <artifactId>jackson-databind</artifactId>
+          <version>2.9.8</version>
+          <scope>test</scope>
+      </dependency>
```

Comment:
Duplicate dependency declaration, and formatting in consistent with the rest of the file.